lib/iex/lib/iex/introspection.ex

```diff
@@ -141,6 +146,18 @@ defmodule IEx.Introspection do
     end
   end
 
+  defp callback_module(mod, fun, arity) do
+    mod.module_info(:attributes)
+    |> Stream.filter_map(&match?({:behaviour, _}, &1), fn {_, l} -> l end)
+    |> Stream.concat()
+    |> Stream.filter(fn(module)->
+      callbacks = module.module_info(:attributes)
+      |> Enum.filter_map(&match?({:callback, _}, &1), fn {_, [{t,_}|_]} -> t end)
+      callbacks[fun] == arity
```

Comment:
If the module exports `[fun: 1, fun: 2]`, this line will match only one of them. Try writing the lines above as a comprehension and just check the result is not an empty list.